pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/elastic/migration/AdditionalKitShippingRetriever.java

```diff
@@ -16,12 +16,14 @@ public class AdditionalKitShippingRetriever extends AdditionalRecordsRetriever<K
     }
 
     static Optional<AdditionalKitShippingRetriever> fromRealm(String realm) {
+/*
```

Comment:
Was this commented out on purpose ?
Same for AdditionalMedicalRecordsRetriever.java and AdditionalOncHistoryDetailsRetriever.java